nusantara/nusa_datasets/tico_19/tico_19.py

```diff
@@ -0,0 +1,282 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+from fnmatch import translate
+import os
+import re
+from pathlib import Path
+from typing import Dict, List, Tuple
+from translate.storage.tmx import tmxfile
+
+import datasets
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{anastasopoulos-etal-2020-tico,
+    title = "{TICO}-19: the Translation Initiative for {CO}vid-19",
+    author = {Anastasopoulos, Antonios  and
+      Cattelan, Alessandro  and
+      Dou, Zi-Yi  and
+      Federico, Marcello  and
+      Federmann, Christian  and
+      Genzel, Dmitriy  and
+      Guzm{\'a}n, Franscisco  and
+      Hu, Junjie  and
+      Hughes, Macduff  and
+      Koehn, Philipp  and
+      Lazar, Rosie  and
+      Lewis, Will  and
+      Neubig, Graham  and
+      Niu, Mengmeng  and
+      {\"O}ktem, Alp  and
+      Paquin, Eric  and
+      Tang, Grace  and
+      Tur, Sylwia},
+    booktitle = "Proceedings of the 1st Workshop on {NLP} for {COVID}-19 (Part 2) at {EMNLP} 2020",
+    month = dec,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2020.nlpcovid19-2.5",
+    doi = "10.18653/v1/2020.nlpcovid19-2.5",
+}
+"""
+
+# We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LANGUAGES = ["ind", "ara", "spa", "fra", "hin", "por", "rus", "zho", "eng"]
+_LOCAL = False
+_SUPPORTED_LANG_PAIRS = [
+    ("ind", "ara"), ("ind", "spa"), ("ind", "fra"), ("ind", "hin"), ("ind", "por"), ("ind", "rus"), ("ind", "zho"),
```

Comment:
In the paper, in the Table4, there are `ind -> eng` pair that they acquire from HelsinkiNLP OPUS-MT. Could you maybe help to check whether this pair can also be loaded?

Reply:
I think the HelsinkiNLP OPUS-MT in Table 4 refers to the MT model, not the dataset. It seems like they use the readily available MT systems (HelsinkiNLP OPUS-MT is one of them) and then test them on their dataset. 

They didn't provide `ind -> eng` language pairs (and `x -> eng`) on [their website](https://tico-19.github.io/memories.html). The [predefined test and validation split](https://tico-19.github.io/data/tico19-testset.zip) are also only provided in `eng -> x`. I guess they just simply reverse the language direction for testing the `x -> eng` language pairs (i.e., use `eng -> x` dataset in opposite direction), but I'm not really sure about it since they didn't explicitly state this in their paper (please correct me if I'm wrong). If we assume that this is correct, then I think we can load `ind -> eng` by using `eng -> ind` in opposite direction. What do you think?

Reply:
Hi @rifkiaputri thank you for elaborating this clearly.

Yes, they somehow didn't complete the table you've mentioned [here](https://tico-19.github.io/memories.html) with all the language pairs logically available. I think they're aiming just to not confuse readers with duplicate links on the page, and that some languages didn't have another pair than from/to English (e.g. Dinka).

I agree with you, I do think that they simply reverse the `eng -> x` to `x -> eng` and didn't provide the reversed link there just to keep the redundancy low. Not only `idn -> eng` is not available there, but `ara -> eng` is also unavailable, but `eng-ara` is there.

TL, DR: Yes, I agree with your suggestion. Thank you @rifkiaputri 👍🏻

Reply:
Hi @bryanwilie! I'm sorry for the slow response. The remaining split has been implemented.

Reply:
No worries. Thank you @rifkiaputri for contributing!